### PR TITLE
New 1.96.0 features blog post

### PR DESCRIPTION
**Pull request by kc-adawson**

### CODE REVIEW

File: _posts/2022-08-17-kubecost-version-1-96-0-feature-highlights.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+![Custom Rightsizing Profiles in 1.96](/assets/images/2022-08-17-kubecost-version-1-96-0-feature-highlights/2.png)
+
+
+## Modes for selecting "Single Aggregation" vs. "Multi Aggregation" when building Asset and Allocation Reports.
+
+In previous releases, we updated the "Aggregation" panel within Cost Allocation to allow for both single and multi aggregation. This was useful when you needed to see allocation reports like "Namespace by Cluster", and "Tag by Namespace" 
+
+In the previous Kubecost versions, toggling between Single and Multi aggregation was not immediatly clear to the user. In 1.96, we've added a clear UI toggle allowing users to pick Single vs Multi aggreation. This makes the process of creating reports easier and gives more clarity to what is container within the Cost Allocation report. 
+
+![Aggregation in 1.96](/assets/images/2022-08-17-kubecost-version-1-96-0-feature-highlights/3.png)
```
Comment (robfaraj) [inline _posts/2022-08-17-kubecost-version-1-96-0-feature-highlights.md:56]: Is it more clear to  offer an animated gif here? I use recordit if you need a tool to easily make one

**Review by robfaraj — COMMENTED** [review]
Overall looks great! Thanks for doing this. I added a few minor changes. We can have Brent give it a once over as well from a grammar standpoint.

File: _posts/2022-08-17-kubecost-version-1-96-0-feature-highlights.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+## What's new in Kubecost 1.96:
+
+- Alerts for asset budget and Cloud Usage Reports
+- Custom profile options to request sizing
+- Modes for selecting "Single Aggregation" vs. "Multi Aggregation" when building Asset and Allocation Reports.
+
+
+## Alerts for 'Asset Budget' and 'Cloud Reports'.
+
+Kubecost 1.96 introduces 2 new Alert types under the Alerting section: "Asset Budget" alerts, and "Cloud Report" alerts. Both of these new alerts can help users get visibility into their out of cluter spend.
```
[the remark this answers is not in the document]
Comment (robfaraj) [reply _posts/2022-08-17-kubecost-version-1-96-0-feature-highlights.md:30]: Both of these can help users get visibility into their out-of-cluster spend.

[the remark this answers is not in the document]
Comment (srpomeroy) [reply _posts/2022-08-17-kubecost-version-1-96-0-feature-highlights.md:30]: SP: cluter shoudl be cluster. But would be fixed with Robs edits.

File: _posts/2022-08-17-kubecost-version-1-96-0-feature-highlights.md [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+tags:
+  - Kubecost
+  - Container Rightsizing
+  - Cost Alerts
+  - Cost Reporting
+---
+
+# Kubecost version 1.96.0 - New Feature Highlights
+
+We've just released the newest generally available version of Kubecost (1.96)! This new update contains a whole bunch of visual and under the hood improvements to the Kubecost platform. In this blog, I'll highlight a few of the new updates you can find after updating to 1.96. To the see the full release notes, check out our [GitHub Releases page](https://github.com/kubecost/cost-analyzer-helm-chart/releases).
```
[the remark this answers is not in the document]
Comment (srpomeroy) [reply _posts/2022-08-17-kubecost-version-1-96-0-feature-highlights.md:18]: https://github.com/kubecost/cost-analyzer-helm-chart/releases/tag/v1.96.0

File: _posts/2022-08-17-kubecost-version-1-96-0-feature-highlights.md (hunk 1 above)
Comment (kc-adawson) [reply _posts/2022-08-17-kubecost-version-1-96-0-feature-highlights.md:56]: Added gif, hows that look? Should I remove the screenshot all together?

**Pull request merged**